src/Console/Report/FixReport/TextReporter.php

```diff
@@ -83,17 +83,18 @@ private function getDiff(bool $isDecoratedOutput, string $diff): string
         return PHP_EOL.$diffFormatter->format($diff).PHP_EOL;
     }
 
-    private function getFooter(int $time, int $memory, bool $isDryRun): string
+    private function getFooter(int $time, int $changed, int $files, int $memory, bool $isDryRun): string
     {
         if (0 === $time || 0 === $memory) {
             return '';
         }
 
         return PHP_EOL.sprintf(
-            '%s all files in %.3f seconds, %.3f MB memory used'.PHP_EOL,
-            $isDryRun ? 'Checked' : 'Fixed',
-            $time / 1000,
-            $memory / 1024 / 1024
-        );
+                '%s %d files in %.3f seconds, %.3f MB memory used'.PHP_EOL,
+                $isDryRun ? 'Checked' : sprintf('Fixed %d of', $changed),
+                $files,
+                $time / 1000,
+                $memory / 1024 / 1024
+            );
```

Comment:
Indentation should not change here.

Reply:
I'm changing that (again it's a thing with my IDE configuration, I'm sorry for that), but maybe there should be some PHP-CS-Fixer configured somewhere to find things like this? ;)

Reply:
I think #4885 will fix that.